airbyte-cdk/python/airbyte_cdk/sources/streams/http/auth/token.py

```diff
@@ -32,3 +33,12 @@ def __init__(self, tokens: List[str], auth_method: str = "Bearer", auth_header:
 
     def get_auth_header(self) -> Mapping[str, Any]:
         return {self.auth_header: f"{self.auth_method} {next(self._tokens_iter)}"}
+
+
+class BasicHttpAuthenticator(TokenAuthenticator):
```

Comment:
I tried not using the deprecated `TokenAuthenticator` with twilio, but I wasn't able to authenticate using the requests_native `TokenAuthenticator` even though the request headers were the same

Reply:
do you know why you weren't able to auth? it seems fishy. Generally I think we should use the requests native auth where possible